sqlmesh/core/renderer.py

```diff
@@ -203,10 +208,12 @@ def _cache_key(
         start: t.Optional[TimeLike] = None,
         end: t.Optional[TimeLike] = None,
         execution_time: t.Optional[TimeLike] = None,
-    ) -> t.Tuple[datetime, datetime, datetime]:
+        evaluating: t.Optional[bool] = None,
```

Comment:
evaluating bool false

Reply:
kept this one because we pass `kwargs.get("evaluating")` which can be `None`